### PR TITLE
Add a ApiCommandHandler which fetch all registered CommandHandlers so that user can get all available CommandHandlers by request /api

### PR DESCRIPTION
**Pull request by all4you**
### Describe what this PR does / why we need it

Add a ApiCommandHandler which fetch all registered CommandHandlers so that user can get all available CommandHandlers by request /api

### Does this pull request fix one issue?

No. Only add new features

### Describe how you did it

Create a new CommandHandler called ApiCommandHandler

### Describe how to verify it

1. Run Application in sentinel-demo-command-handler module
2. Request 127.0.0.1:8080/init to init the Sentinel and CommandCenter
3. Request 127.0.0.1:8719/api to get all the available CommandHandlers

### CODE REVIEW

File: sentinel-transport/sentinel-transport-common/src/main/java/com/alibaba/csp/sentinel/command/annotation/CommandMapping.java [hunk 1]
```diff
@@ -30,4 +26,7 @@
 public @interface CommandMapping {
 
     String name();
+
+    String desc();
```
Comment (sczyh30) [inline sentinel-transport/sentinel-transport-common/src/main/java/com/alibaba/csp/sentinel/command/annotation/CommandMapping.java]: Maybe it's better to be optional (default empty)?

Comment (all4you) [reply sentinel-transport/sentinel-transport-common/src/main/java/com/alibaba/csp/sentinel/command/annotation/CommandMapping.java]: Define the desc to be not empty aims to force user create CommandHandler with a detailed description. If set default empty when user get the api list they don't know what the exactly usage of the api. So I would prefer it not be default empty.

Comment (sczyh30) [reply sentinel-transport/sentinel-transport-common/src/main/java/com/alibaba/csp/sentinel/command/annotation/CommandMapping.java]: Okay, it's reasonable.

**Comment by all4you** [comment]
> Maybe it's better to add an example to show how to register a customized `CommandHandler` via SPI in `sentinel-demo-command-handler` as it's a demo module.

done!

**Comment by sczyh30** [comment]
Nice, thanks for contributing!

**Pull request merged**